Automattic/sensei
# Add a button to the quiz block that opens the quiz settings

Continuation of #4345, also closes #4345
Fixes #4391

### Changes proposed in this Pull Request

* Adds a button in the quiz block which opens the quiz settings.

### Testing instructions

* Open the lesson editor.
* Click the newly introduced 'Quiz settings' button and observe that the quiz block is selected and its settings are opened.

### Screenshot

<img width="692" alt="Screenshot on 2022-01-06 at 00-49-27" src="https://user-images.githubusercontent.com/1612178/148300833-72261a22-827f-4154-b9c4-b631e9225cf1.png">
